main.js

```diff
@@ -140,8 +131,8 @@ export function getCredentialStatus({credential, statusPurpose} = {}) {
   const credentialStatuses = _getStatuses({credential});
   const result = credentialStatuses.filter(
     credentialStatus => _validateStatus({credentialStatus})).find(
-    // check for matching `type` (legacy) or `statusPurpose`
-    cs => cs.type === expectedType || cs.statusPurpose === statusPurpose);
+    // check for matching `statusPurpose`
+    cs => cs.statusPurpose === statusPurpose);
```

Comment:
Is this the right resolution to the problem from before? What happens if the type doesn't match `StatusList2021Entry` (assuming that's the right type here)? Is this checked another way? It seems like maybe we're missing test coverage for this.

Reply:
@dlongley can you pls say more about what the expected behavior for that above case should be and also, is `statusPurpose` a required property of `credentialStatus`?

Reply:
`statusPurpose` is only a required property of `credentialStatus` when the credential status type is `StatusList2021Entry`. A VC may have multiple credential status entries, however, some that may not have that type. We want to ignore those. We should not be throwing on unknown status types. Does that explain everything?

Reply:
@dlongley yeah, makes sense. thanks

Reply:
Updated and added test here [946bb2e](https://github.com/digitalbazaar/vc-status-list/pull/6/commits/946bb2ef6996b1da6750340f5f4acfd0f2523431).